plugins/mcreator-localization/lang/texts.properties

```diff
@@ -848,4 +848,36 @@ elementgui.command.warning.empty_string=Command can't be empty string
 elementgui.fuel.burn_time=Fuel burn time (power): 
 elementgui.fuel.properties=Fuel properties
 modelement.painting=Painting
-modelement.painting.description=Painting is a picture that you can put on your walls using the Painting item
\ No newline at end of file
+modelement.painting.description=Painting is a picture that you can put on your walls using the Painting item
+elementgui.tool.stay_in_grid_when_crafting=Check to enable
+elementgui.tool.damage_on_crafting=Check to enable
+elementgui.tool.model_normal=Normal
+elementgui.tool.has_glow=Check to enable
+elementgui.tool.event_right_clicked_air=When right clicked in air (player loc.))
+elementgui.tool.event_crafted=When item is crafted/smelted
+elementgui.tool.event_right_clicked_block=When right clicked on block (hand loc.))
+elementgui.tool.event_block_destroyed=When block destroyed with tool
+elementgui.tool.event_entity_hit_with=When living entity is hit with tool
+elementgui.tool.event_in_inventory_tick=When tool in inventory tick
+elementgui.tool.event_in_hand_tick=When tool in hand tick
+elementgui.tool.event_stopped_using=On player stopped using
+elementgui.tool.event_swings=When entity swings item
+elementgui.tool.tool_texture=Tool texture
+elementgui.tool.item_model=<html>Item model:<br><small>Select the item model to be used. Supported: JSON, OBJ
+elementgui.tool.tool_3d_model=Tool 3D model
+elementgui.tool.tool_special_information=<html>Special information about the tool:<br><small>Separate entries with comma, to use comma in description use \\
+elementgui.tool.name_in_gui=Name in GUI:

```

Comment:
I already a line for Name in GUI at the top of the "modelement....". Use it instead of adding a new one.

Reply:
Fixed!